src/main/java/org/zoo/Animals.java

```diff
@@ -1,4 +1,4 @@
-package org.example;
+package org.zoo;
 public abstract class Animals extends EarthCreatures {
```

Comment:
Why is this class abstract? I don't see any abstract methods in it

Reply:
fixed